src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -64,46 +65,51 @@
 #include <OutputReportPredefined.hh>
 #include <PlantComponent.hh>
 #include <PlantUtilities.hh>
+#include <Psychrometrics.hh>
 #include <ReportSizingManager.hh>
 #include <UtilityRoutines.hh>
-#include <WaterToWaterHeatPumpEIR.hh>
+#include <PlantLoopHeatPumpEIR.hh>
 #include "CurveManager.hh"
 
 namespace EnergyPlus {
-    namespace EIRWaterToWaterHeatPumps {
+    namespace EIRPlantLoopHeatPumps {
 
-        bool getInputsWWHP(true);
-        std::vector<EIRWaterToWaterHeatPump> eir_wwhp;
-        std::string static const __EQUIP__ = "EIRWaterToWaterHeatPump"; // NOLINT(cert-err58-cpp)
+        bool getInputsPLHP(true);
+        std::vector<EIRPlantLoopHeatPump> eir_plhp;
```

Comment:
I really shouldn't have done snake case here when I first added `eir_wwhp`.  If you have more changes to make, feel free to make it `eirPlhp` or even better just `heatPumps`.